src/App.tsx

```diff
@@ -1,10 +1,57 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/Todolist/TodoList';
+import { TodosFilter } from './components/TodosFilter/TodosFilter';
+import { Statuses } from './types/Common';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 9953;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [status, setStatus] = useState<Statuses>(Statuses.ALL);
+
+  const visibleTodos = todos.filter((todo) => {
+    switch (status) {
+      case Statuses.ACTIVE:
+        return !todo.completed;
+
+      case Statuses.COMPLETED:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((fetchedTodos: Todo[]) => {
+        setTodos(fetchedTodos);
+      })
+      .catch((fetchedError: Error) => {
+        setError(fetchedError?.message ?? 'Something went wrong');
+      });
+  }, []);
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
```

Comment:
```
I am not sure but if you want to save your timer in variable you should do this, but it's just suggestion
    let timerId: number | null = null;
```

Reply:
Sorry, I was in a hurry with the comment, but in general, such timeouts in the browser return numbers, but in the NodeJs environment, they are really Timers. Within this project, the typescript settings work with NodeJs type for timers, so you can leave it as is.